verification/policy_file_processor.go

```diff
@@ -0,0 +1,334 @@
+package verification
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/notaryproject/notation-go/dir"
+)
+
+var trustPolicyPath string
+
+func init() {
+	trustPolicyPath = dir.Path.TrustPolicy()
```

Comment:
I noticed that there are two functions in package `dir`
```
// TrustPolicyForWrite returns the writable path of trustpolicy.json file
func (p *PathManager) TrustPolicyForWrite(writeLevel WriteLevel) string
```
and
```
// TrustPolicy returns the ready-only path of trustpolicy.json file
func (p *PathManager) TrustPolicy() string
```
Shoud we also pass `WriteLevel` as an argument?

Reply:
Right, looks like https://github.com/notaryproject/notation-go/pull/137 is not merged yet. I'll do the refactoring after it.

Reply:
> Right, looks like #137 is not merged yet. I'll do the refactoring after it.

[PR ](https://github.com/notaryproject/notation-go/pull/134)in dev branch has been merge.  Just a reminder :satisfied:

Reply:
I see, thanks for reminder